core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -3612,19 +3615,23 @@ join sales.emp e on e.deptno = d.deptno and d.deptno not in (4, 6)]]>
         <Resource name="planBefore">
             <![CDATA[
 LogicalProject(EMPNO=[$0], ENAME=[$1], JOB=[$2], MGR=[$3], HIREDATE=[$4], SAL=[$5], COMM=[$6], DEPTNO=[$7], SLACKER=[$8], EMPNO0=[$9], ENAME0=[$10], JOB0=[$11], MGR0=[$12], HIREDATE0=[$13], SAL0=[$14], COMM0=[$15], DEPTNO0=[$16], SLACKER0=[$17])
-  LogicalJoin(condition=[AND(=($16, $7), NOT(OR(=($7, 4), =($7, 6))))], joinType=[inner])
-    LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-    LogicalTableScan(table=[[CATALOG, SALES, EMP]])
+  LogicalFilter(condition=[true])
```

Comment:
Why do all these redundant filters appear? Should we possibly change the value of `config.simplify()` somewhere?

Reply:
Hi @zabetak , it these rule, simplification is not on:
![image](https://user-images.githubusercontent.com/15643702/114866268-f10be800-9e25-11eb-873a-ef4e75586aa0.png)